### PR TITLE
Updating localization validator to v2.0.0

### PR DESCRIPTION
**Pull request by mishra14**

### CODE REVIEW

File: NuGetBuildValidators/common.ps1 [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "{0:HH:mm:ss} +{1:F0}" -f $currentTime, ($currentTime - $lastTime).TotalSeconds
+}
+
+$Global:LastTraceTime = Get-Date
+
+Function Format-ElapsedTime($ElapsedTime) {
+    '{0:F0}:{1:D2}' -f $ElapsedTime.TotalMinutes, $ElapsedTime.Seconds
+}
+
+Function Invoke-BuildStep {
```
Comment (rohit21agrawal) [inline NuGetBuildValidators/common.ps1]: you don't need the teamcity stuff anymore

Comment (mishra14) [reply NuGetBuildValidators/common.ps1]: Build scripts need updating

File: NuGetBuildValidators/NuGetValidator/NuGetValidator.csproj [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <PackageReference Include="NuGet.Common" Version="4.0.0" PrivateAssets="All" />
+    <PackageReference Include="ILRepack" Version="2.0.13" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NuGetValidator.Localization\NuGetValidator.Localization.csproj" />
+    <ProjectReference Include="..\NuGetValidator.Utility\NuGetValidator.Utility.csproj" />
+  </ItemGroup>
+
+  <Target Name="PublishAndIlMerge" BeforeTargets="_IntermediatePack">
```
Comment (rohit21agrawal) [inline NuGetBuildValidators/NuGetValidator/NuGetValidator.csproj]: you can remove this dependency on _IntermediatePack, and start using a property $(BeforePack) to make your target execute before pack.

File: NuGetBuildValidators/NuGetValidator.Localization/NuGetValidator.Localization.csproj [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <ItemGroup>
+    <PackageReference Include="ILRepack" Version="2.0.13" PrivateAssets="All" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NuGetValidator.Utility\NuGetValidator.Utility.csproj" />
+  </ItemGroup>
+
+  <Target Name="PublishAndIlMerge" BeforeTargets="_IntermediatePack">
```
Comment (rohit21agrawal) [inline NuGetBuildValidators/NuGetValidator.Localization/NuGetValidator.Localization.csproj]: same here, use BeforePack property isntead

**Pull request merged**